### ISSUE TITLE
a11y > cart sidebar > <label> for form control

### ISSUE THREAD

**Issue by ryanschaefer41** [vigetlabs/EcoCart#76]
Related to #68.

I believe the actual selector needs a `<label>` element also.

From WCAG guidelines: "If a form control does not have a properly associated text label, the function or purpose of that form control may not be presented to screen reader users."

Found these potential solutions to make sure it's accessible to screenreaders:
- a `<label>` element with a for attribute value that is equal to the id of a unique form control
- a `<label>` element that surrounds the form control, does not surround any other form controls, and does not reference another element with its for attribute a non-empty title attribute
- a non-empty aria-labelledby attribute.

**Comment by ryanschaefer41** [comment]
@lwbecker Was this one merged?

**Comment by nellbhealy** [comment]
Yes this was merged
